### PR TITLE
Team2-Signoff-CO2 & NO2 Monitoring Subsystem - Signoff.md

### PR DESCRIPTION
**Pull request by jaredhooker**
Requesting Signoff for CO2 & NO2 Monitoring Subsystem

### CODE REVIEW

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md [hunk 1]
```diff
@@ -0,0 +1,207 @@
...
+
+- **Carbon Dioxide Sensor – MH-Z19B**
+
+  - MH-Z19B needs between 4.5-5.5V and at least 20mA to operate eﬃciently.
+
+  - The sensor must turn on every 5 minutes to measure the amount of carbon
+
+    dioxide present in the greenhouse.
+
+  - Must alert when above or below 400-2000ppm to ensure proper plant growth.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md]: Based on what? Also what does alert mean?

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+- **Arduino Nano 33 BLE**
+
+  - The communication between the sensors and PLC must be wireless. This will save
+
+    time and money on wiring, while also allowing the system to be scalable.
+
+  - The transfer of data between the sensors and the Arduino must be nearly
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md]: What does this mean?

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md [hunk 3]
```diff
@@ -0,0 +1,207 @@
...
+
+    instantaneous for accurate and reliable monitoring of the CO and NO levels in
+
+    the atmosphere.
+
+  - BLE (Bluetooth Low Energy) is lower in cost than the classic Bluetooth as BLE is
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md]: And it's not compatible with many Bluetooth devices... probably ought to check into that.

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md [hunk 4]
```diff
@@ -0,0 +1,207 @@
...
+
+- **Nitrogen Dioxide Sensor – MIKROE-3700**
+
+  - MIKROE-3700 needs between 3.3-5V to operate eﬃciently.
+
+  - The sensor must turn on every 5 minutes to measure the amount of nitrogen
+
+    dioxide present in the greenhouse.
+
+  - Must alert when above Nitrogen Dioxide level exceeds 5 ppm.
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md]: Based on what?

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md [hunk 5]
```diff
@@ -0,0 +1,207 @@
...
+
+- **Silicon Conformal Coating**
```
Comment (JesseTNRoberts) [inline Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md:62]: Why?

**Review by JesseTNRoberts — COMMENTED** [review]
The formatting of this document looks very bad. Address this.

Show all the pins of your devices. I don't think this is sufficient for the connections. Why is there a resistive divider to power a device?

File: Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md (hunk 4 above)
Comment (Noah-Jones37) [reply Documentation/Signoffs/CO2 & NO2 Monitoring Subsystem - Signoff.md]: The CDC's guidelines, which is source 2.

**Review by JesseTNRoberts — COMMENTED** [review]
There's no analysis of the design showing that the circuit will turn on the system when needed and will have the required battery life. I don't see anything analyzing whether this will have the needed sensitivity.

**Review by JesseTNRoberts — COMMENTED** [review]
We discussed that the constraint regarding 400-2000ppm seems to be incorrect.

**Comment by Noah-Jones37** [comment]
> We discussed that the constraint regarding 400-2000ppm seems to be incorrect.

After further research, we concluded that the values are correct. A source has been cited to prove this.

**Pull request merged**